scripts/convert_to_csv.py

```diff
@@ -1,40 +1,35 @@
 import pyreadr
-import glob, os
+from pathlib import Path
 
 
 errors = []
+need_zip = [
+    "ncdc_normals.rda",
+]
 
-for fn in glob.glob("../data/rda_files/*.rda"):
+rda_dir = Path("../data/rda_files")
+csv_dir = Path("../data/csv_files")
+errors_file = Path("..") / "data" / "errors.txt"

```

Comment:
Personal preference, but I'd rather have the script output errors to `stderr` rather than write a file. If there is a file, there's no way to know if it is from the last run or some previous run, and you need to be concerned with cleaning it up... If the errors are just emitted to `stderr` then they can capture them optionally on their own, if desired. You can print to `stderr` with
```python
print("stuff", file=sys.stderr)
```

Reply:
Does this mean deleting the `errors.txt` file altogether?

Reply:
That's my suggestion, it's (IMO) just an extra hassle. By emitting to `stderr` it gives the user options (including redirecting to a file, if that's what they want) without forcing things